sdk/react/ooyalaSkinPanelRenderer.js

```diff
@@ -90,7 +91,9 @@ OoyalaSkinPanelRenderer.prototype.renderErrorScreen = function() {
     <ErrorScreen
       error={this.skin.state.error}
       localizableStrings={this.skin.props.localization}
-      locale={this.skin.props.locale}>
+      locale={this.skin.props.locale}
+      isAudioOnly={this.skin.state.screenType === SCREEN_TYPES.ERROR_SCREEN_AUDIO}
```

Comment:
We have a `contentType` in `OoyalaSkin` for this purpose I think.

Reply:
The problem is that on Android when there is no internet connection there is no info about content type from s bridge.
So changing here to contentType will do nothing and audio contentType or screenType will.be false.

Logic to make it work is implemented here:
I can change it to contentType, but anyways this state must be send from Android's bridge.

OoyalaSkinBridgeListener.prototype.onError = function(e) {
  Log.log("Error received");
  this.skin.setState({
    screenType: e.screenType == SCREEN_TYPES.AUDIO_SCREEN ?
      SCREEN_TYPES.ERROR_SCREEN_AUDIO : SCREEN_TYPES.ERROR_SCREEN,
    error: e
  });
};